cpp/src/parquet/types.cc

```diff
@@ -345,15 +345,19 @@ std::shared_ptr<const LogicalType> LogicalType::FromConvertedType(
     case ConvertedType::DATE:
       return DateLogicalType::Make();
     case ConvertedType::TIME_MILLIS:
```

Comment:
Can you add a reference to https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#deprecated-time-convertedtype ?

Reply:
done

Reply:
It would be good to add the parity for timestamp type as well: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#deprecated-timestamp-convertedtype, just in case people are looking for timestamp type only.

Reply:
done